_posts/2021-04-10-newpipe-0.21.1-released.md

```diff
@@ -0,0 +1,39 @@
+﻿---
+layout: post
+title: "NewPipe 0.21.1 released: EU Consent Cookie Hotfix"
+short: "NewPipe 0.21.1 released"
+date: 2021-04-10 XX:XX:XX + 1:00
+categories: [pinned, release]
+author: Opusforlife & Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+This is a hotfix release to bypass the new YouTube consent page shown to users in the European Union.
+
+The current workaround is not guaranteed to work forever. It may be deprecated soon. You could say this is just a "warm fix".
```

Comment:
> "warm fix"

Pretty confusing neologism. I'd remove the last sentence and create just one paragraph as intro to improve the reading flow.

Reply:
Done